website/blog/2022-08-01-aluminium.md

```diff
@@ -0,0 +1,355 @@
+    <td>New features</td>
+    <td align="center">6</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/<num>?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Scala 3] Type completion
+- [Scala 3] Auto complete and import extension methods
+- [Scala 3] ConvertToNamedParameters code action
+- [Scala 3] ScaladocCompletion for Scala3
+- [Scala 3] Add interpolation completions
+- [Scala 2] Automatically import types in string interpolations
+- More stable code navigation across subprojects
+- Code Action documents
+- Support Scala 3.2.0-RC3, Scala 3.2.0-RC2
+
+and a lot of bugfixes!
+
+## [Scala 3] Type Competion
+
+## [Scala 3] Auto complete and import extension methods
+
+Scala3 introduced `extension methods` that allows defining new methods to the existing types.
+
+However, previously, Metals couldn't auto-complete extension methods, and developers have to find an appropriate extension methods from workspace and import it by hand. It is quite time-consuming.
+
+Now, Metals provides auto-completion for the extension methods and auto-import them!
+
+## [Scala 3] ConvertToNamedParameters code action
+
+[Metals 0.11.7 added `ConvertToNamedParameters` code action to Scala2](https://scalameta.org/metals/blog/2022/07/04/aluminium#scala-2-add-converttonamedarguments-code-action).
+
+Thanks to the contribution by [@jkciesluk](https://github.com/jkciesluk), this feature is now available for Scala3!
+
+![convert-to-named](https://i.imgur.com/9i7MWoQ.gif)
+
+## [Scala 3] Scaladoc completion
+
+Metals now supports auto completing scaladoc completion in Scala3. Typing `/**`, you can auto-complete scaladoc template for the method, class, and etc!
+
+![scala-doc-completion](https://i.imgur.com/MEJUXr3.gif)
+
+## [Scala 3] Interpolation completion
+
+## [Scala 2] Auto import types in string interpolations
+
+## More stable code navigation across subprojects
+
+Previously, Metals sometimes fail to code navigation such as find-all-references and go-to-defintions across projects.
+
+Now, Metals provides more stable code navigation thanks to the work by [@kpodsiad](https://github.com/kpodsiad)!
+
+If you're interested in technical details, please check out the PR [\#4176](https://github.com/scalameta/metals/pull/4176).
+
+## Code Action documents
+
+Have you ever wondered what kind of refactorings are available in Metals?
+Check out the new document! You can learn the list of code actions with examples.
+https://scalameta.org/metals/docs/codeactions/codeactions
+
+Big thanks to [zmerr](https://github.com/vzmerr) for writing this documentation.
```

Comment:
I'm actually not sure which account should I point to 😅 FYI @vzmerr

Reply:
anyone you prefer😅